hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/authorization/ValidateAccessToken.java

```diff
@@ -40,8 +45,14 @@ public ValidateAccessToken(AuthorizationProperties authorizationProperties, Stri
     @Override
     public void process(Exchange exchange)
             throws Exception {
-        AccessToken accessToken = AccessToken.parse(exchange.getIn().getHeader("Authorization").toString());
-        logger.info(String.format("Access token: %s", accessToken));
+    	String methodName = "process";
+    	String authorizationKey = (String) exchange.getIn().getHeader(AUTH_HEADER_KEY);
+    	if(authorizationKey==null) {
```

Comment:
I'm a big fan of using Apache Commons Land 3 StringUtils (http://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html) for all String comparisons. It's got every type of String manipulation and conditions you could ever want. Something like StringUtils.isBlank() or StringUtils.isEmpty() would probably be a better choice than a direct null check.